packages/protocol/contracts/tokenvault/ERC20Vault.sol

```diff
@@ -121,12 +121,6 @@ contract ERC20Vault is
         _;
     }
 
-    /// @notice Initializes the contract with the address manager.
-    /// @param addressManager Address manager contract address.
-    function init(address addressManager) external initializer {
-        EssentialContract._init(addressManager);
```

Comment:
Shall we still keep this `init` method and call `AuthorizableContract.init(addressManager)` here?